docs/source/resources/scyllaclusters/basics.md

```diff
@@ -61,7 +61,7 @@ metadata:
 spec:
   repository: {{imageRepository}}
   version: {{imageTag}}
-  agentVersion: 3.5.1
+  agentVersion: {{agentVersion}}
```

Comment:
Some substitutions are missing in the following files:
- resources/scyllaclusters/nodeoperations/restore.md
- resources/scyllaclusters/basics.md
- resources/scyllaclusters/multidc.md
- resources/scylladbclusters/scylladbclusters.md

Reply:
indeed, missed a few. Fixed